example/example_parallel.py

```diff
@@ -8,10 +8,13 @@
 
 from autosklearn.classification import AutoSklearnClassifier
 from autosklearn.constants import *
+from autosklearn.util.backend import BackendContext
 
 tmp_folder = '/tmp/autosklearn_parallel_example_tmp'
 output_folder = '/tmp/autosklearn_parallel_example_out'
 
+backend_context = BackendContext(tmp_folder, output_folder, True, True)
```

Comment:
I think the backend should not be used in the examples. Can you replace this by a simple os.remove?